src/Microsoft.AspNetCore.Razor.Tools/Roslyn/CommandLineUtilities.cs

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Roslyn.Utilities
```

Comment:
It might be good to add a note here that we basically copied this verbatim. I don't think there's much value in us trying to refactor or change this code.